plugins/MauticFocusBundle/EventListener/ReportSubscriber.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace MauticPlugin\MauticFocusBundle\EventListener;
+
+use Mautic\ReportBundle\Event\ReportBuilderEvent;
+use Mautic\ReportBundle\Event\ReportGeneratorEvent;
+use Mautic\ReportBundle\ReportEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ReportSubscriber implements EventSubscriberInterface
```

Comment:
All new classes should be `final` by default unless there is a reason not to. It helps a lot with potential BC breaks later on where you don't have to worry about some plugin extending this subscriber when changing a public method signature for example.
```suggestion
final class ReportSubscriber implements EventSubscriberInterface
```